config/install/search_api.index.primary.yml

```diff
@@ -51,9 +51,9 @@ third_party_settings:
       specific_languages:
         en: '0'
       use_universal_collation: false
-id: primary
+id: Primary
```

Comment:
What is the motivation for uppercasing the id? Wouldn't it be more typical to keep the id all lower, and uppercase the first character of the name?